### PR TITLE
19782 shipment cancellation date validation be int quick fix

### PR DESCRIPTION
**Pull request by KonstanceH**
## Summary

This PR prevents the TOO from cancelling any shipment that's 'actualPickupdate' is on or after the request date.

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### How to test

1. Make a HHG 
2. Login as SC to edit and submit move 
4. Login as TOO and  approve and send move
5. Go to the 'Move Task Order' tab and try to cancel the move
6. Confirm you're able to delete the move without an actualPickupdate

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

## Screenshots
Screenshot 2024-06-07 at 10 23 00 AM

### CODE REVIEW

File: src/constants/userRoles.js [hunk 1]
```diff
@@ -19,7 +19,6 @@ export const adminOfficeRoles = [
   { roleType: 'services_counselor', name: 'Services Counselor' },
   { roleType: 'prime_simulator', name: 'Prime Simulator' },
   { roleType: 'qae_csr', name: 'Quality Assurance and Customer Support' },
-  { roleType: 'headquarters', name: 'Headquarters' },
```
Comment (taeJungCaci) [inline src/constants/userRoles.js:22]: I think this needs to be not deleted. Prob from merge conflict.

**Review by taeJungCaci — CHANGES_REQUESTED** [review]

File: src/constants/userRoles.js (hunk 1 above)
Comment (KonstanceH) [reply src/constants/userRoles.js]: yea I pulled it in from a conflict. Just removed it and pushed it up

**Review by taeJungCaci — APPROVED** [review]
LGTM!

**Review by brooklyn-welsh — APPROVED** [review]
LGTM ✅

**Pull request merged**